### ISSUE TITLE
Add the total recorded/validated count on the contribute page

### ISSUE THREAD

**Issue by zecakeh** [mozilla/voice-web#1293]
On the new contribute page, we only see the countdown for the current 5 sentences.

I set myself a goal of 100 sentences a day, so it's easier to follow with the total count rather than remember how many "sessions" I have done.

**Comment by Gregoor** [comment]
Fair enough! We're also considering making goal setting part of the app (once we have profiles). But short term we then maybe should consider showing the tallies of daily/total contributions.

cc @m-branson

**Comment by m-branson** [comment]
Thanks for this feedback @zecakeh & for the cc @Gregoor, apologies for delay as I've been away. Indeed we are looking to include goal tracking with the launch of profiles (coming soon, thanks for patience as we get there). Eventually we would like to evolve this into personal goal setting once we've gathered more research & tested the initial goal experience.

We could introduce the stats counter to the contribution experience as a holistic overview of how numbers are improving as a daily session progresses, though I am curious @zecakeh if you'd instead prefer a session based count rather than a view of your overall count? Or is it both?

_I'm looking to think this through before we jump to a solution, which is why we didn't add the counter by default in the first place._ :)

**Comment by zecakeh** [comment]
@m-branson Session based count would be enough on the Contribute page, but the overall count should definitely be accessible somewhere else (profile or stats page maybe).

**Comment by m-branson** [comment]
@Gregoor @LRSaunders this sounds like a great candidate for A/B testing. Let's capture this (session based counter vs. full stat counter vs. no counter in the contribution experience) and get it on a roadmap. Happy to do so, just need to be pointed to the A/B doc -- slack works for that if easier.

Thanks @zecakeh for the quick reply, we'll work toward getting some data around this to prove out which stats prove best in the contribution flow. As for overall stats, those will always be available via the header & after we launch the new profiles overall stats will be available on a dashboard + the header when logged in. 👍
